README.md

```diff
@@ -1,28 +1,59 @@
-![Nodezoo](https://raw.githubusercontent.com/rjrodger/nodezoo-web/to-redux/client/assets/img/logo-nodezoo.png)
+![nodezoo](https://raw.githubusercontent.com/rjrodger/nodezoo-web/to-redux/client/assets/img/logo-nodezoo.png)
 
-# nodezoo-NPM-UPDATE
-Nodezoo.com micro-service handling npm data.. Please see the [main repo][] for more details.
+## nodezoo-npm-update
+Nodezoo.com micro-service handling npm data. Please see the [main repo][] for more details.
 
 - __Sponsor:__ [nearForm][]
+- __Lead:__ [Richard Rodger][Lead]
 
+## Install
+1. clone this repo into a root _/nodezoo_ folder.
+2. run `npm install`
 
+## Starting
+To start simply run,
+```
+npm start
+```
+### Tagging and Logs
+To tag your service and set up logs simply pass the relevant switches on start,
+
+```
+npm start -- --seneca.options.tag=nodezoo-github --seneca.log.all
+```
+
+## Inbound Messages
 
+This micro-service listens for the following messages:
 
-# Running
+  * _role:npm,task:registry_subscribe_
+  * _role:npm,task:process_modules_
+  * _role:npm,task:download_modules_
 
-```sh
-$ node npm-service.js --seneca.options.from=dev.options.js --seneca.log.all
+## Outbound Messages
+
+This micro-service emits no outbound messages.
+
+## Running with Curl
+
+Any of the messages above can be run using curl in the following format in the commanding
+```
+curl -d '{"role":"npm","task":"registry_subscribe"}' http://localhost:44005/act
 ```
 
```

Comment:
Add > __Note:__ Ports are assigned automatically, please check the logs for the correct port to use.